### PR TITLE
Added GitOps Con Videos

### PR DESCRIPTION
**Pull request by vanessaabankwah**
Signed-off-by: Vanessa Abankwah <96237690+vanessaabankwah@users.noreply.github.com>

### CODE REVIEW

File: content/en/resources.yaml [hunk 1]
```diff
@@ -1,4 +1,25 @@
...
+     - youtube: Wgp04xTNqq4
+    title: "Automate Your SLO Validation with Prometheus & Flagger - Sanskar Jaiswal & Kingdon Patrick Barrett"
+    date: 2022-10-25
+    - youtube: cYcmXCJ2tLU
+    title: "Simplifying Edge Deployments Using EMCO and GitOps - Igor DC & Adarsh Vincent Chittilappilly, Intel"
+    date: 2022-10-25
+   - youtube: h8G3LM9uIHk
+    title: "Toward Full Adoption of GitOps and Best Practices at RingCentral - Ivan Anisimov & Tamao Nakahara"
+    date: 2022-10-25
+       - youtube: 7dOKxIYEtvY
```
Comment (Arhell) [inline content/en/resources.yaml:20]: Need to move to the left

File: content/en/resources.yaml [hunk 2]
```diff
@@ -1,4 +1,25 @@
...
+    title: "How to Achieve (Actual) GitOps with Terraform and Flux - Priyanka Ravi, Weaveworks"
+    date: 2022-10-25
```
Comment (dholbach) [inline content/en/resources.yaml:22]:
```suggestion
     - youtube: G1Mt4KE4Dao
    title: "Flux + Observability: Featuring Prometheus Operator and Pixie - Somtochi Onyekwere, Weaveworks"
    date: 2022-10-25
    - youtube: Ums3Q9kMPd8
    title: "GitOps with Flux and OCI Registries - Soulé Ba & Scott Rigby, Weaveworks"
    date: 2022-10-25
    - youtube: jRil9H1NhZI
    title: "Complete DR of Stateful Workloads, PVs and CSI Snapshots via Flux and Vault OSS - Kingdon Barrett"
    date: 2022-10-25
     - youtube: Wgp04xTNqq4
    title: "Automate Your SLO Validation with Prometheus & Flagger - Sanskar Jaiswal & Kingdon Patrick Barrett"
    date: 2022-10-25
    - youtube: cYcmXCJ2tLU
    title: "Simplifying Edge Deployments Using EMCO and GitOps - Igor DC & Adarsh Vincent Chittilappilly, Intel"
    date: 2022-10-25
   - youtube: h8G3LM9uIHk
    title: "Toward Full Adoption of GitOps and Best Practices at RingCentral - Ivan Anisimov & Tamao Nakahara"
    date: 2022-10-25
       - youtube: 7dOKxIYEtvY
    title: "How to Achieve (Actual) GitOps with Terraform and Flux - Priyanka Ravi, Weaveworks"
    date: 2022-10-25 
```

Comment (dholbach) [inline content/en/resources.yaml:22]:
```suggestion
  - youtube: G1Mt4KE4Dao
    title: "Flux + Observability: Featuring Prometheus Operator and Pixie - Somtochi Onyekwere, Weaveworks"
    date: 2022-10-25
  - youtube: Ums3Q9kMPd8
    title: "GitOps with Flux and OCI Registries - Soulé Ba & Scott Rigby, Weaveworks"
    date: 2022-10-25
  - youtube: jRil9H1NhZI
    title: "Complete DR of Stateful Workloads, PVs and CSI Snapshots via Flux and Vault OSS - Kingdon Barrett"
    date: 2022-10-25
  - youtube: Wgp04xTNqq4
    title: "Automate Your SLO Validation with Prometheus & Flagger - Sanskar Jaiswal & Kingdon Patrick Barrett"
    date: 2022-10-25
  - youtube: cYcmXCJ2tLU
    title: "Simplifying Edge Deployments Using EMCO and GitOps - Igor DC & Adarsh Vincent Chittilappilly, Intel"
    date: 2022-10-25
  - youtube: h8G3LM9uIHk
    title: "Toward Full Adoption of GitOps and Best Practices at RingCentral - Ivan Anisimov & Tamao Nakahara"
    date: 2022-10-25
  - youtube: 7dOKxIYEtvY
    title: "How to Achieve (Actual) GitOps with Terraform and Flux - Priyanka Ravi, Weaveworks"
    date: 2022-10-25 
```

**Review by dholbach — APPROVED** [review]
Thanks a lot @vanessaabankwah !

**Pull request merged**